kubernetes/loculus/values.yaml

```diff
@@ -1989,7 +1989,7 @@ images:
     pullPolicy: Always
   lapis:
     repository: "ghcr.io/genspectrum/lapis"
-    tag: "0.5.14"
+    tag: "v0.5.19"
```

Comment:
The `v` is a bit surprising here.

Reply:
This is because the branch I created is called `v0.5.19` to tag it as `0.5.19` I would either need to create a new branch or log into the docker container repo to tag it

Reply:
maybe we should also pin the Sha just in case - like we do for silo now

Reply:
@anna-parker  you thought it wasn't worth adding the sha?